airbase/pom.xml

```diff
@@ -359,6 +359,9 @@
                                     <exclude>org.eclipse.jetty:jetty-alpn-java-server</exclude>
                                 </excludes>
                             </enforceBytecodeVersion>
+                            <requirePluginVersions>
+                                <message>Providing the plugin version explicitly is important for build stability</message>
```

Comment:
"reproducible builds" ?

Reply:
Reworded it as "Providing the plugin version explicitly makes the builds reproducible"